internal/proposals/migration-hooks/migration-hooks-proposal.md

```diff
@@ -1,6 +1,6 @@
 # JobManager (Migration Logic)
 
-This PoC investigates a valid design for the new __JobManager__, which is needed to enable a fully-automated Kyma deploy. It will be used to configure the cluster and the components during the deployment of Kyma. The terms "Deployment" and "Deploy" are used in the context of installing Kyma on an empty cluster, or to upgrade Kyma from an older to a newer version.
+This PoC investigates a valid design for the new __JobManager__, which is needed to enable a clean automated Kyma deploy. It will be used to configure the cluster and the components during the deployment of Kyma. The terms "Deployment" and "Deploy" are used in the context of installing Kyma on an empty cluster, or to upgrade Kyma from an older to a newer version.
```

Comment:
Is it `JobManager` or `jobManager`? 
You use the first here and in the title, and the second in the rest of the document. Please adjust it :)

Reply:
Since package names always start lower case, I will correct it to `jobManager`